### ISSUE TITLE
Option to edit release instead of adding new release

### ISSUE THREAD

**Issue by hasezoey** [kellnerd/harmony#73]
I have recently started using harmony, whereas before i had been doing stuff completely manually.
This project is great for adding new releases, but i would like a option to augment / extend existing releases, similar to `Import to MusicBrainz`(`musicbrainz.org/release/add`) but not adding a whole new release.

From what i can tell, nothing like this exists in harmony yet and i dont know if musicbrainz supports importing to `musicbrainz.org/release/MBID/edit`.
(i have also checked harmony `/release/actions`, but also cant find it there)

**Comment by kellnerd** [comment]
Yes, it is possible to seed `/release/<mbid>/edit` but I am not aware of any tool which makes use of this so far.
I've already teased the idea to support this on the forum thread, but never made a ticket for this, so thank you for doing that!

My [forum comment](https://community.metabrainz.org/t/harmony-music-metadata-aggregator-and-musicbrainz-importer/698641/80?u=kellnerd) for completeness:
> Another great feature which now becomes doable, and with which I have already experimented, is the possibility to update existing releases with additional/modified data. But this requires more work, both on Harmony’s side and [on MBS itself ](https://tickets.metabrainz.org/browse/MBS-13688), so don’t get too excited about this yet.
> It should be easier to start with only seeding additional external links first, everything else probably needs to be configurable (i.e. keep original value or overwrite), and updating the tracklist itself is not currently possible at all.

**Comment by kellnerd** [comment]
> It should be easier to start with only seeding additional external links first, everything else probably needs to be configurable

Updating the external links is now possible (https://github.com/kellnerd/harmony/commit/eeafbd6244f358f68debc74cd8a1ee240d49ede0) with today's update, MB is doing the deduplication for us.

Adding support for other release properties (except the tracklist) is now trivial, but I want to expose an easy way to select the preferred provider first. Otherwise it would be quite annoying for the user having to revert unwanted changes in the release editor by hand.
Currently the preferred provider is implicitly the one for the looked up URL or the first ticked checkbox. Instead we will want MB to be the default source of truth, with other providers only filling the gaps by default.

**Comment by whyman** [comment]
What's the latest on this? I've a bunch of releases without UPC or links that I want to tidy up, so running them through Harmony would save me a ton of effort
